etc/kayobe/kolla/globals.yml

```diff
@@ -2,6 +2,9 @@
 # Add extra Kolla global configuration here.
 rabbitmq_monitoring_user: "monitoring"
 
+# Although Kolla-Ansible provides full support for Fernet tokens, it still
+# defaults to UUID.  This setting overrides K-A and brings it in-line with
+# Keystone's default.
 keystone_token_provider: "fernet"
 keystone_tag: 6.0.0.0
```

Comment:
Shall we remove this and below to allow keystone and magnum to use the new images?